include/openthread/dataset.h

```diff
@@ -213,6 +213,17 @@ typedef struct otOperationalDatasetComponents
     bool mIsChannelMaskPresent : 1;      ///< TRUE if Channel Mask is present, FALSE otherwise.
 } otOperationalDatasetComponents;
 
+/**
+ * This struct represents a Thread Dataset timestamp.
```

Comment:
nit: follow existing doc style
```suggestion
 * This structure represents a Thread Dataset timestamp.
```